README.rst

```diff
@@ -4,4 +4,4 @@ sherlockml
 .. image:: https://travis-ci.org/ASIDataScience/sherlockml.svg?branch=master
     :target: https://travis-ci.org/ASIDataScience/sherlockml
```

Comment:
This and the line above should point at the faculty Travis project.

Reply:
good catch.